app/src/main/java/fr/free/nrw/commons/customselector/database/UploadedDao.kt

```diff
@@ -0,0 +1,112 @@
+package fr.free.nrw.commons.customselector.database
+
+import androidx.room.*
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.runBlocking
+import java.util.*
+
+/**
+ * UploadedStatusDao for Custom Selector.
+ */
+@Dao
+abstract class UploadedStatusDao {
+
+    /**
+     * Insert into uploaded status.
+     */
+    @Insert( onConflict = OnConflictStrategy.REPLACE )
+    abstract suspend fun insert(uploadedStatus: UploadedStatus)
+
+    /**
+     * Update uploaded status entry.
+     */
+    @Update
+    abstract suspend fun update(uploadedStatus: UploadedStatus)
+
+    /**
+     * Delete uploaded status entry.
+     */
+    @Delete
+    abstract suspend fun delete(uploadedStatus: UploadedStatus)
+
+    /**
+     * Get All entries from the uploaded status table.
+     */
+    @Query("SELECT * FROM uploaded_table")
+    abstract suspend fun getAll() : List<UploadedStatus>
```

Comment:
Do we really need this one?

Reply:
Just for debugging, I'll clear it off as soon as I finish working on this table.